VimeoNetworking/Sources/VimeoClient.swift

```diff
@@ -233,56 +231,56 @@ final public class VimeoClient
                 }
             }
             
-            if request.cacheFetchPolicy == .CacheOnly
+            if request.cacheFetchPolicy == .cacheOnly
             {
                 return RequestToken(path: request.path, task: nil)
             }
             
-        case .NetworkOnly, .TryNetworkThenCache:
+        case .networkOnly, .tryNetworkThenCache:
             break
         }
         
-        let success: (NSURLSessionDataTask, AnyObject?) -> Void = { (task, responseObject) in
-            dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0)) {
+        let success: (URLSessionDataTask, Any?) -> Void = { (task, responseObject) in
+            
+            DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
Yeah, that's a good point. I just spend some time looking into this and I'm not sure how to work around it. These APIs appear to be iOS 10+ / tvOS 10+, but the old `dispatch_async` APIs aren't available anymore. Does that mean that once we're done with the Swift 3 conversion we can't support iOS 8 or 9? Is there a workaround?

@mikew-personal What did you end up doing?